event-handlers/src/test/java/no/sikt/nva/nvi/events/evaluator/EvaluateNviCandidateHandlerTest.java

```diff
@@ -523,6 +553,203 @@ void shouldCreateNewCandidateEventWhenAffiliationAreNviInstitutions() throws IOE
     }
 
+    @Test
+    void shouldIdentifyCandidateWithUnverifiedNviCreators() throws IOException {
+        // Define input data and mock external services
+        var unverifiedContributor =
+            ExpandedContributor.from(DEFAULT_VERIFIED_CONTRIBUTOR)
+                               .withId(null)
+                               .withVerificationStatus(null)
+                               .build();
+        var publication =
+            ExpandedInputResourceGenerator.from(DEFAULT_INPUT_PUBLICATION)
+                                          .withContributors(List.of(unverifiedContributor))
+                                          .build();
+        var fileUri =
+            s3Driver.insertFile(
+                UnixPath.of(publication.identifier().toString()),
+                publication.toJsonString());
+        mockCristinResponseAndCustomerApiResponseForNviInstitution(okResponse);
+
+        // Define expected response
+        var unverifiedNviCreators =
+            List.of(
+                new UnverifiedNviCreator(
+                    unverifiedContributor.contributorName(),
+                    unverifiedContributor.affiliationIds()));
+
+        var expectedCandidate =
+            getDefaultExpectedCandidateBuilder()
+                .withPublicationId(publication.id())
+                .withPublicationBucketUri(fileUri)
+                .withInstitutionPoints(emptyList())
+                .withTotalPoints(ZERO)
+                .withNviCreators(emptyList())
+                .withUnverifiedNviCreators(unverifiedNviCreators)
+                .build();
+        var expectedEvaluatedMessage =
+            CandidateEvaluatedMessage.builder().withCandidateType(expectedCandidate).build();
+
+        // Call handler and parse response
+        var event = createEvent(new PersistedResourceMessage(fileUri));
+        handler.handleRequest(event, context);
+        var messageBody = getMessageBody();
+
+        // Assert that response matches exactly what we expect
+        assertEquals(expectedEvaluatedMessage, messageBody);
+    }
+
+    @Test
+    void shouldExcludeUnverifiedCreatorsFromInstitutionPoints() throws IOException {
+        // Define input data and mock external services
+        var unverifiedContributor =
+                ExpandedContributor.from(DEFAULT_VERIFIED_CONTRIBUTOR)
+                                   .withId(null)
+                                   .withVerificationStatus(null)
+                                   .build();
+        var contributors = List.of(DEFAULT_VERIFIED_CONTRIBUTOR, unverifiedContributor);
+        var publication =
+                ExpandedInputResourceGenerator.from(DEFAULT_INPUT_PUBLICATION)
+                        .withContributors(contributors)
+                        .build();
+        var fileUri =
+                s3Driver.insertFile(
+                        UnixPath.of(publication.identifier().toString()),
+                        publication.toJsonString());
```

Comment:
Extract to method, duplicated in 4 tests, maybe something like `insertPublicationToS3(publication)`

Reply:
Thanks, done!